### PR TITLE
Update plugin python script to correctly handles multi-line env vars

### PR DESCRIPTION
**Pull request by mahmoudajawad**
Current script fails to handle output where any of the env vars value is multi-line (includes a newline).

### CODE REVIEW

File: plugins/index.vim [hunk 1]
```diff
@@ -2,13 +2,33 @@ python3 << EOF
...
-    output = pipe.communicate()[0].decode('utf8').splitlines()
-    env = dict((line.split("=", 1) for line in output))
-    os.environ.update(env)
+    output = pipe.communicate()[0].decode("utf8").splitlines()
 
+    def update_os_environ(line: str) -> None:
+        env_var, env_val = line.split("=", 1)
+        os.environ[env_var] = env_val
+        print("Updateing env var: " + env_var)
+
```
Comment (sansyrox) [inline plugins/index.vim]: Can you move this(function definition) above the `if "VIRTUAL_ENV"` statement?

**Review by sansyrox — COMMENTED** [review]
Hey @mahmoudajawad .

Thank you for the PR and apologies for the late review. Can you please address the inline changes and then this PR should be good for merge.

File: plugins/index.vim (hunk 1 above)
Comment (mahmoudajawad) [reply plugins/index.vim]: Applied in second commit

**Comment by mahmoudajawad** [comment]
Hello, @sansyrox.
I missed your comment earlier. Just updated the file. Have a look at it, please.

**Comment by mahmoudajawad** [comment]
@sansyrox, a quick reminder to merge this.

**Comment by sansyrox** [comment]
Hey @mahmoudajawad ,
Sorry, I missed your update😅

Good job ✨

**Comment by mahmoudajawad** [comment]
Thanks. Have good day.

**Pull request merged**